README.md

```diff
@@ -72,6 +72,24 @@ Divvy allows users to enter expenses as they are incurred, and calculates the be
 15. Click the "Show" button for "App Secret". Enter your Facebook password to display the value. Copy the shown value and into the `.env.js` file as the `FACEBOOK_SECRET`.
 16. With the dev server running, navigating to `http://localhost:3000/auth/facebook` redirects the browser to Facebook and ask for app authorization. Once authorized, the browser redirects to `http://localhost:3000/auth/facebook/callback` with a `code` query param (logged on the server). The browser then redirects to the application root.
 
+## Deployment (Heroku)
+
```

Comment:
Assumes user has a developer account with Heroku. Accounts are free to create for up to five projects.